src/styles/applications/_departmentlist.scss

```diff
@@ -1,5 +1,24 @@
 .dg_department_group {
+  .dg_card {
+    flex-wrap: nowrap;
+    overflow-wrap: break-word;
```

Comment:
the flex-wrap is the one thing Im unsure about. However if this stays the word-break is overridden and spills out. Not sure if Im missing something there or not.